src/Controllers/PrioritisationController.php

```
@@ -35,13 +38,34 @@ class PrioritisationController
     protected Auth $auth;
     protected Database $db;
     protected array $config;
-    public function __construct(Request $request, Response $response, Auth $auth, Database $db, array $config)
-    {
+    private \Closure $geminiFactory;
+
+    public function __construct(
+        Request $request,
+        Response $response,
+        Auth $auth,
+        Database $db,
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)